src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -42,7 +42,7 @@ public void isSamePerson() {
 
         // name differs in case, all other attributes same -> returns false
```

Comment:
Now that a name that differs in case now should return true, maybe this comment can be changed accordingly?

Reply:
ohya correct i missed this out!